### PR TITLE
[April Fools] Official NT Greytide Initiative

### PR DESCRIPTION
**Pull request by TheOneAndMightyRed**
## About The Pull Request

Only for Test Merging during april fools

Adds a bunch of resprited versions of items in the game to get that authentic "greytide" appearance.

Every item is the equivalent of another item stats wise.
grey jumpsuit = TGMC Uniform
armor vest = PAS-11 (without lights)
insulated gloves = Engineer combat insulated gloves
helmet = M10 helmet (actually a hardhat with the same light_range as the PAS-11)
jackboots = marine combat boots
gas mask = default gas mask
radio = default mainship headset so peeps can still see the ranks and follow their SL
Ancient Toolbox = Machete without it's scabbard.
[ERT] Super Ancient Toolbox = Machete but you can strap it onto your hand like you can with the TL-172
[ERT] riot shield = TL-172
Each greyshirt also gets secHUDglasses (<-- only marines), a default grey backpack and a megaphone.

Toolboxes fit on both belt slot and suit storage slot and have actual mob sprites to indicate that.

Adds a greytide ERT with a special toolbox and riot shield they can strap onto their hands. ERT has double the chance of spawning as a normal ERT does, because april fools.

Everything is shitcoded together but it's only for like a day so technical debt is of no concern.
(Unless you actually merge this you fool!)

[Vending starter kit](https://i.imgur.com/5zp0W8P.png)
[What you look like](https://i.imgur.com/eIbs9IK.png)
[Items in a stack when vending the starter kit as a marine](https://i.imgur.com/tMdRupZ.png)
^Engis get welding goggles instead of sechuds, medics and SLs get their health hud so they can actually do their job.

If people don't like playing as a greyshirt, they always have the option to get a normal uniform and jaeger suit from the surplus vendor, instead of having to stick with the meme shit.

Essentially people will become PAS-11 machete wielding maniacs here to save humanity from big bad aliens.

I SUCK AT MAKING DESCRIPTIONS SO PLEASE LEAVE FEEDBACK THANK.

## Why It's Good For The Game

April Fools content for like a day.

## Changelog
:cl:
add: TGMC is contractually obliged to offer every marine the chance to get armed up with weaponry generously provided by the official Nanotrasen Greytide Initiative. Actually using their gear will make you a fool.
/:cl:

### CODE REVIEW

**Comment by XSlayer300** [comment]
Oh boi, multiple april fools events time.

File: code/datums/emergency_calls/pmc.dm [hunk 1]
```diff
@@ -52,3 +52,41 @@
...
+//April fools - greytide ERT
+
+/datum/emergency_call/greytide
+	name = "NT Greytide Escalation Squad"
+	probability = 50 //Only one day so let them greytide
+	shuttle_id = "distress_pmc"
+
+
+/datum/emergency_call/greytide/print_backstory(mob/living/carbon/human/H)
+	to_chat(H, "<B>After leaving space station 13 in irreparable disarray, you decided to join the ranks of the official greytide department of Nanotrasen.</b>")
```
Comment (XSlayer300) [inline code/datums/emergency_calls/pmc.dm:65]:
```suggestion
	to_chat(H, "<B>After leaving Space Station 13 in irreparable disarray, you decided to join the ranks of the official Greytide department of Nanotrasen.</b>")
```

File: code/game/objects/items/weapons/blades.dm [hunk 2]
```diff
@@ -181,6 +181,35 @@
 	attack_speed = 12
 	w_class = WEIGHT_CLASS_BULKY
 
+//April fools toolbox. Same state of the machete but a seperate item because it shouldn't go into the machete scabbard.
+/obj/item/weapon/claymore/toolbox
+	name = "Ancient Toolbox"
```
Comment (XSlayer300) [inline code/game/objects/items/weapons/blades.dm:186]:
```suggestion
	name = "ancient toolbox"
```

File: code/game/objects/items/weapons/blades.dm [hunk 3]
```diff
@@ -181,6 +181,35 @@
...
+	icon_state = "toolbox"
+	item_state = "toolbox"
+	force = 60
+	attack_speed = 12
+	w_class = WEIGHT_CLASS_BULKY
+	attack_verb = list("robusted", "horizontals", "tides")
+
+//Used by ERT
+/obj/item/weapon/claymore/toolbox/nodrop
+	name = "Super Ancient Toolbox"
```
Comment (XSlayer300) [inline code/game/objects/items/weapons/blades.dm:197]:
```suggestion
	name = "super ancient toolbox"
```

**Comment by ilikesaying** [comment]
**GREYTIDE WORLDWIDE!!**

**Pull request closed without merging**